app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java

```diff
@@ -0,0 +1,65 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.annotation.NonNull;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.matcher.BoundedMatcher;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+
+import static android.support.test.internal.util.Checks.checkNotNull;
+import static org.mozilla.focus.fragment.FirstrunFragment.FIRSTRUN_PREF;
+
+
+public final class AndroidTestUtils {
+
+    public static void beforeTest() {
+        final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        if (context == null) {
+            return;
+        }
+        final SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
+        if (preferences != null) {
+            final SharedPreferences.Editor editor = preferences.edit();
+            if (editor != null) {
+                editor.putBoolean(FIRSTRUN_PREF, true).commit();
+            }
+        }
+        final Settings settings = Settings.getInstance(context);
+        if (settings != null) {
+            settings.setShareAppDialogDidShow();
+            settings.setRateAppDialogDidShow();
+        }
+    }
+
+    public static Matcher<View> atPosition(final int position, @NonNull final Matcher<View> itemMatcher) {
```

Comment:
Suggest to have a separate class(ex: RecyclerViewMatcher) for this helper function